src/System.ServiceModel.Federation/src/System/ServiceModel/Federation/IWSTrustChannelContract.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IdentityModel.Tokens;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Protocols.WsTrust;
+
+namespace System.ServiceModel.Federation
+{
+    /// <summary>
+    /// A service contract that defines the Issue request.
+    /// </summary>
+    [ServiceContract]
+    [ComVisible(false)]
+    public interface IWSTrustChannelContract : IWSTrustContract
```

Comment:
We don't have this on any of our other interfaces. Is this here for a specific reason or just because of existing coding patterns?

Reply:
Existing patterns.

Reply:
removed [ComVisible(false)] in: ab9b3c1cb00f558dcd6ac6f3b02663ace5e25750